### PR TITLE
Invitations using fake credential services

### PR DESCRIPTION
**Pull request by rchodava**

### CODE REVIEW

**Review by seveibar — APPROVED** [review]

File: src/lib/database/store.ts [hunk 1]
```diff
@@ -665,28 +738,122 @@ const initializer = immer<Database>((set, get) => ({
...
+      )
+    }
+
+    return get().enrollment_automations.filter(
+      (automation) =>
+        automation.user_identity_id === client_session.user_identity_id,
+    )
   },
 
   getInvitation(params) {
```
Comment (mikelittman) [inline src/lib/database/store.ts:859]: do we still need this method now?

**Review by mikelittman — APPROVED** [review]
had a question re: existing `getInvitation` method but otherwise looks great, can always clean that up later if it's not needed anymore

File: src/lib/database/store.ts (hunk 1 above)
Comment (rchodava) [reply src/lib/database/store.ts:859]: used by the `internal/phone/get_invitation` endpoint I think

Comment (mikelittman) [reply src/lib/database/store.ts:859]: ill prob clean that up in a subsequent pr, probably better to keep it a little dry

**Pull request merged**